src/app/components/case/WorkflowForm/WorkflowForm.tsx

```diff
@@ -3,6 +3,7 @@
 import { Fields } from "app/components/shared/Form/ScaffoldFields"
 import ConfirmScaffoldForm from "app/components/shared/ConfirmScaffoldForm/ConfirmScaffoldForm"
 import useNavigateWithFlashMessage from "app/state/flashMessages/useNavigateWithFlashMessage"
+import { useParams } from "@reach/router"
```

Comment:
Interesting. Didn't know about that hook. But think it might be more logical to add a `camundaTaskId` property to `WorkflowForm`?

Reply:
You mean like this?
```
<WorkflowForm
        id={ id }
        fields={ fields }
        postMethod={ execPost }
        camundaTaskId={ taskId }
      />
```

Reply:
Yes

Reply:
Will do